__tests__/app.test.js

```diff
@@ -49,18 +50,105 @@ describe("GET: /api/topics", () => {
       .get("/api/topics")
       .expect(200)
       .then((results) => {
-        console.log("model results");
-        console.table(results.body);
         expect(results.body).toEqual(testTopics);
       });
   });
   test("404: Returns 404 when invalid request", () => {
     return request(app)
       .get("/api/topicsINVALID")
       .expect(404)
-      .then((results)=>{
-          console.log("HERE",results.status);
+      .then((result) => {
+        expect(result.body.msg).toBe("404 - path not found");
       });
   });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("400 article_id provided is not in database", () => {
```

Comment:
this should be a 404 as the article 999 could exist but it doesn't yet 
then test for a 400 for something that's an invalid data type i.e articles/dog